assets/src/edit-story/components/panels/test/textStyle.js

```diff
@@ -491,7 +493,7 @@ describe('Panels/TextStyle', () => {
       expect(pushUpdate).toHaveBeenCalledWith({ lineHeight: 1.5 });
     });
 
-    it('should set lineHeight to empty', () => {
+    it('should clear line height if set to empty', () => {
```

Comment:
It doesn't really clear it though, in practice, it seems to remain, the value.

That is the case for most of the other values as well -- the value remains instead of clearing / setting to default.

The only exception seems to be letter-spacing -- that's cleared and set to 0 if the input is emptied (or well, it doesn't seem to be possible to really empty the field). Should the letter-spacing also remain then for consistency?

Reply:
I will fix the line height and letter spacing input so we can clear the input (so it will match all other fields) and then will check the tests to match that.

Reply:
@miina any idea why letter-spacing is live updated? (as opposed to all other fields). I'm wondering if it was that intentional?

Reply:
Hmm, probably an oversight, can't think of a reason right now why that would be the only one to have different logic.

Reply:
Maybe related to rich text supporting partial letter-spacing for partial text selection?

The behavior for the user should still be the same though -- if it's not an easy change then we could follow up separately.

Reply:
I'm leaving the letter-spacing input as it is, full dive needed here since we might face some unwanted effects. https://github.com/google/web-stories-wp/issues/2636